com.unity.toonshader/Documentation~/Basic.md

```diff
@@ -2,9 +2,6 @@
 
 **Three Color Map and Control Map Settings** provide basic cell shading settings in the **Unity Toon Shader**. These settings allow you to control the rendering of light and shadow areas independent from the actual light color. UTS allows detailed control whether the directional light color effects on materials. Please refer to [Scene Light Effectiveness Settings](SceneLight.md).
 
-<img src="images/InspectorThreeColorAndControlMapSettings.png" width="573">
```

Comment:
I see that we are removing a couple of images like this one in this PR.
Any particular reason ?